packages/vendor-dev/package.json

```diff
@@ -18,16 +18,13 @@
   "files": [
     "lib",
     "scss",
-    "babel.preset.js",
-    "eslint.extends.js"
+    "babel.preset.js"
   ],
   "scripts": {
     "lint": "eslint ."
   },
   "dependencies": {
-    "@theforeman/find-foreman": "^4.15.0",
     "@theforeman/vendor-core": "^4.15.0",
-    "babel-plugin-module-resolver": "^3.2.0",
-    "eslint-import-resolver-alias": "^1.1.2"
+    "babel-plugin-module-resolver": "^3.2.0"
```

Comment:
why keep "babel-plugin-module-resolver" in vendor-dev, this is still needed?

Reply:
I was removing the packages I stopped using in the `vendor-dev` but actually, we can safely remove the vendor-dev entirely as it is not used anymore.